core/whistle_media-1.0.0/src/wh_media_map.erl

```diff
@@ -362,7 +362,10 @@ get_map(?WH_MEDIA_DB = Db, PromptId) ->
             #media_map{id=MapId
                        ,account_id=Db
                        ,prompt_id=PromptId
-                       ,languages=wh_json:new()
+                       ,languages=wh_json:set_value(wh_media_util:default_prompt_language()
+                                                    ,<<"/", ?WH_MEDIA_DB/binary, "/", (wh_media_util:default_prompt_language())/binary, "%2F", PromptId/binary>>
```

Comment:
Actually, I've got the same problem on 3.18, 3.19 and master (this weekend), so it looks more like a Kazoo misconfiguration instead of code difference. And since this problem appears not ony with installations of mine, probably a kind of failover should be done. May be it should be inserted  into ets also